Resources/config/schema/routing-1.0.xsd

```diff
@@ -83,7 +83,6 @@
         <xsd:attribute name="enabled" type="xsd:boolean" />
         <xsd:attribute name="manager-name" type="xsd:string" />
         <xsd:attribute name="content-basepath" type="xsd:string" />
-        <xsd:attribute name="use-sonata-admin" type="enable_auto" />
```

Comment:
Well, I don't think so. I think we should maybe even remove the version. This is why:

 * We don't host the XSD, so IDE's can never fetch it by URL;
 * Symfony will always validate against the local XSD, which is the latest;
 * Renaming this to 2.0 means everyone has to update their schemaLocations to `routing-2.0.xsd`, even though they didn't use this attributed. Or we have to ship both 1.0 and 2.0, but we don't support both, we only support 2.0;
 * [Symfony core doesn't do versioning either](https://github.com/symfony/symfony/issues/20635).